File: launchdarkly/ruby/src/PostAIConfigExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

ai_config_post = LaunchDarklyClient::AIConfigPost.new
ai_config_post.key = "key"
ai_config_post.name = "name"
ai_config_post.description = ""
ai_config_post.tags = [
    "tags",
    "tags",
]

begin
    response = LaunchDarklyClient::AIConfigsBetaApi.new.post_ai_config(
        "beta", # ld_api_version
        "projectKey_string", # project_key
        ai_config_post,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AIConfigsBetaApi#post_ai_config: #{e}"
end

File: launchdarkly/php/src/PostAIConfigExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

$ai_config_post = (new LaunchDarkly\Client\Model\AIConfigPost())
    ->setKey("key")
    ->setName("name")
    ->setDescription("")
    ->setTags([
        "tags",
        "tags",
    ]);

try {
    $response = (new LaunchDarkly\Client\Api\AIConfigsBetaApi(config: $config))->postAIConfig(
        ld_api_version: "beta",
        project_key: "projectKey_string",
        ai_config_post: $ai_config_post,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling AIConfigsBetaApi#postAIConfig: {$e->getMessage()}";
}
